inlang/source-code/versioned-interfaces/plugin/src/interface.ts

```diff
@@ -19,7 +19,7 @@ import type { ExternalProjectSettings, ProjectSettings } from "@inlang/project-s
  */
 export type NodeishFilesystemSubset = Pick<
 	NodeishFilesystem,
-	"readFile" | "readdir" | "mkdir" | "writeFile" | "watch"
+	"readFile" | "readdir" | "mkdir" | "writeFile" | "watch" | "rm" | "stat"
```

Comment:
I repeat the question here. Which plugin needs access to `rm` and `stat`? 

The subset exists to keep APIs that plugins use under control and ease a migration to the upcoming lix fs API

Reply:
resolved.